packages/react-data-grid/src/Cell.tsx

```diff
@@ -199,36 +199,40 @@ export default class Cell extends React.PureComponent<Props> implements CellRend
     return null;
   }
 
-  renderCellContent() {
-    let cellContent;
-    const { value, column, height, tooltip, isScrolling, expandableOptions, cellMetaData, rowIdx } = this.props;
-    const Formatter = column.formatter;
-    const cellProps: FormatterProps<unknown> = {
-      rowIdx,
-      value,
-      isScrolling,
-      column,
+  getFormatterProps(): FormatterProps<unknown> {
+    return {
+      rowIdx: this.props.rowIdx,
+      value: this.props.value,
+      isScrolling: this.props.isScrolling,
+      column: this.props.column,
       row: this.getRowData(),
       dependentValues: this.getFormatterDependencies()
```

Comment:
This would log a warning for every single cell if `getRowMetaData` was truthy, even though it's not used with the SimpleCellFormatter.